1-3.md

```diff
@@ -97,25 +102,26 @@ Approximately 30 to 45 minutes.
 
 {% include excol.html type="middle" %}
 
-Introduce other benefits of accessibility, such as technical or financial. Use the resource [Developing an Accessibility Business Case for your Organization (Archived)](https://www.w3.org/WAI/business-case/archive/).
```

Comment:
I find this very brief and unclear. Tried to expand a little more and provide some guidance.